VibeSomeMauiBro/Views/CollectionPage.xaml.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections.ObjectModel;
+using VibeSomeMauiBro.Models;
+using VibeSomeMauiBro.Services;
+
+namespace VibeSomeMauiBro.Views;
+
+public partial class CollectionPage : ContentPage
+{
+    private readonly ICatService _catService;
+    
+    public ObservableCollection<Cat> LikedCats { get; } = new();
+
+    public CollectionPage(ICatService catService)
```

Comment:
Can this use C# primary constructiors for the `_catService`:

* https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/primary-constructors

Add some info about this to `github-instructions.md` as well.